api-samples/declarativeNetRequest/url-redirect/README.md

```diff
@@ -0,0 +1,17 @@
+# chrome.declarativeNetRequest - URL Blocker
+
+This sample demonstrates how to use the [chrome.declarativeNetRequest](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/) API to redirect requests.
+
+## Overview
+
+Once installed, any requests made in the main frame to the following URLs will be redirected:
```

Comment:
```suggestion
Once this extension is installed, any requests made in the main frame to the following URLs will be redirected:
```

Reply:
This seems like it breaks my advice on omitting needless words, but 'once installed' is a dangling modifier.